### ISSUE TITLE
Input Help: NVDA doesn't read Arabic Characters

### ISSUE THREAD

**Issue by a2hsh** [nvaccess/nvda#11725]
### Steps to reproduce:
1. Change keyboard layout to Arabic (Saudi Arabia) - Arabic (101).
2. Switch Input Help mode on, using NVDA+1.
3. Type any characters.
### Actual behavior:
NVDA reads the English character instead of the Arabic one, although the keyboard layout is set to Arabic.
### Expected behavior:
NVDA should respect the keyboard layout and read the Arabic character.
#### NVDA installed/portable/running from source:
Installed.
#### NVDA version:
2020.3 beta 3
#### Windows version:
Windows 10, Version 2004 (OS Build 19041.508)
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
Tried setting Windows language to Arabic, as well as NVDA language to Arabic with the same results.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Tried NVDA version 2020.1 and 2020.2 with the same result
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes
### Notes
> I'm an instructor for the blind on using the computer with NVDA, this issue proposes a challenge to new users who try to practice using the keyboard.

All the best.

**Comment by mohdshara** [comment]
Hi.

This is actually a duplicate of #7317 
You may want to follow the conversation there.

Thanks
Mohammed.

**Comment by feerrenrut** [comment]
Thanks @mohdshara I'll close this as a duplicate. If this is a mistake, please let us know.
